code/client/src/core/ui/player_debug.cpp

```diff
@@ -0,0 +1,165 @@
+#include "player_debug.h"
+#include <external/imgui/wrapper.h>
+#include <imgui.h>
+
+#include "sdk/c_player_teleport_module.h"
+#include "sdk/entities/c_player_2.h"
+
+#include "game/helpers/controls.h"
+
+namespace MafiaMP::Core::UI {
+    PlayerDebug::PlayerDebug() {}
+
+    void PlayerDebug::Update(bool *isVisible) {
+        const auto pActivePlayer = Game::Helpers::Controls::GetLocalPlayer();
+        if (!pActivePlayer)
+            return;
+
+        ImGui::Begin("Player debug", isVisible);
+
+        auto position = pActivePlayer->GetPos();
+        if (ImGui::DragFloat3("Pos", (float *)&position, 0.1f, -2000.0f, 2000.0f)) {
+            pActivePlayer->SetPos(position);
+        }
+
+        auto dir = pActivePlayer->GetDir();
+        if (ImGui::DragFloat3("Dir", (float *)&dir, 0.01f, -1.0f, 1.0f)) {
+            pActivePlayer->SetDir(dir);
+        }
+
+        auto rot = pActivePlayer->GetRot();
+        if (ImGui::DragFloat4("Rot", (float *)&rot, 0.01f, -1.0f, 1.0f)) {
+            pActivePlayer->SetRot(rot);
+        }
+
+        /**
+         * TODO: Scale is float or Vector3 ?
+         */
+        // auto scale = pActivePlayer->GetScale();
+        // if (ImGui::SliderFloat4("Scale", (float *)&scale, 0.0f, 1.0f)) {
+        //     // pActivePlayer->SetScale(scale);
+        // }
+
+        // float scale = pActivePlayer->GetScale();
+        // if (ImGui::SliderFloat("Scale", &scale, 0.0f, 1.0f)) {
+        //     pActivePlayer->SetScale(scale);
+        // }
```

Comment:
Either implement or remove to avoid polluting codebase

Reply:
Code removed since we have a dedicated issue for: https://github.com/MafiaHub/MafiaMP/issues/49